terraform/aws/infrastructure.tf

```diff
@@ -109,8 +109,8 @@ resource "aws_security_group" "secgroup" {
   vpc_id = local.vpc_id
 
   tags = {
-    Name      = "${local.deployment_name}-sg"
-    Workspace = local.deployment_name
+    name      = "${local.deployment_name}-sg"
+    workspace = local.deployment_name
```

Comment:
In the `terraform/aws/modules` dir, all module (hana_node etc) `main.tf` files also have those defined, like

```
Name      = "${var.common_variables["deployment_name"]}-drbd-subnet-${count.index + 1}"
    Workspace = var.common_variables["deployment_name"]
    ```
    are they fine with the capital letters there?

Reply:
> In the `terraform/aws/modules` dir, all module (hana_node etc) `main.tf` files also have those defined, like
> 
> ```
> Name      = "${var.common_variables["deployment_name"]}-drbd-subnet-${count.index + 1}"
>     Workspace = var.common_variables["deployment_name"]
> ```
> 
> are they fine with the capital letters there?

no, let em fix it